### ISSUE TITLE
Compatible with Slim 3 ?

### ISSUE THREAD

**Issue by Rudloff** [christianklisch/slim-minify#7]
Does this middleware work with Slim 3?
I get this error:

```
Fatal error: Class 'Slim\Middleware' not found in /home/pierre/www/alltube/vendor/christianklisch/slim-minify/src/Slim/Middleware/Minify.php on line 34
```

**Comment by guiscaranse** [comment]
I am also getting this error

**Comment by christianklisch** [comment]
closed with version 0.5.0 #9 #
